dScripts/02_server/Map/NT/NtCombatChallengeServer.cpp

```diff
@@ -43,7 +43,7 @@ void NtCombatChallengeServer::OnFireEventServerSide(Entity* self, Entity* sender
 
 
 void NtCombatChallengeServer::OnMessageBoxResponse(Entity* self, Entity* sender, int32_t button, const std::u16string& identifier, const std::u16string& userData) {
-	if (identifier == u"PlayButton" && button == 1) {
+	if (identifier == u"PlayButton" && button == 1 && !self->GetVar<bool>(u"bInUse")) {
```

Comment:
The variable in this case is a network variable, which is separate from the normal once. This will have to use the GetNetworkVar variant on the method.

Reply:
Oh interesting, I must have mis-read the API.  Should be a simple fix :)